packages/klighd-core/tsconfig.json

```diff
@@ -23,5 +23,8 @@
   },
   "include": [
     "src"
-  ]
-}
\ No newline at end of file
+  ],
+  "paths": {
+    "@kieler/klighd-interactive/lib/*": ["../klighd-interactive/lib/*"]
+  }
+}
```

Comment:
What is this change for?

Reply:
I had problems with imports resolution in vs-code and this is what I found as a solution.
Should I remove this change?

Reply:
Also I am somehow not seeing this review diff/conversion in the diff view

Reply:
I see the problem now, I guess this is fine then.